pyproject.toml

```diff
@@ -34,6 +29,7 @@ classifiers = [
     "Topic :: Scientific/Engineering",
 ]
 dependencies = [
+    "pybammsolvers",
```

Comment:
This makes it a non-optional dependency, and is a breaking change as well – please see my comment above.

Reply:
The intention is to make it required, but not default yet